### PR TITLE
add missing cmd `RemoveSwarmNodeCmd` in client

### PR DESCRIPTION
**Pull request by ziminghua**
I want to remove a swarm node but the command not in docker client, please merge this PR as soon as possible. thx

### CODE REVIEW

File: docker-java/src/test/java/com/github/dockerjava/cmd/swarm/RemoveSwarmNodeCmdExecIT.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Remove swarm node cmd test
+ *
+ * @author 訾明华
+ * @since 2021-11-10
+ */
```
Comment (bsideup) [inline docker-java/src/test/java/com/github/dockerjava/cmd/swarm/RemoveSwarmNodeCmdExecIT.java:23]: this is unnecessary
```suggestion
```

File: docker-java/src/test/java/com/github/dockerjava/cmd/swarm/RemoveSwarmNodeCmdExecIT.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+public class RemoveSwarmNodeCmdExecIT extends SwarmCmdIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RemoveSwarmNodeCmdExecIT.class);
+
+    @Test
+    public void testUpdateSwarmNode() throws Exception {
```
Comment (bsideup) [inline docker-java/src/test/java/com/github/dockerjava/cmd/swarm/RemoveSwarmNodeCmdExecIT.java:29]:
```suggestion
    public void testRemoveSwarmNode() throws Exception {
```

File: docker-java/src/test/java/com/github/dockerjava/cmd/swarm/RemoveSwarmNodeCmdExecIT.java (hunk 1 above)
Comment (ziminghua) [reply docker-java/src/test/java/com/github/dockerjava/cmd/swarm/RemoveSwarmNodeCmdExecIT.java]: sorry, it's my idea file template.

**Comment by ziminghua** [comment]
sorry, that header is my idea file template.

**Comment by bsideup** [comment]
@ziminghua the build is failing - could you please have a look?

**Pull request merged**